packages/expo-module-scripts/bin/expo-module-build

```diff
@@ -8,7 +8,7 @@ args=("$@")
 
 # If the command is used like `yarn build plugin`, set the --build option to point to
 # plugin/tsconfig.json
-extra_module_build_types=("plugin" "cli" "utils")
+extra_module_build_types=("plugin" "cli" "utils" "scripts")
```

Comment:
I _think_ this should be fine. Note that other packages that have a `scripts` directory will now compile scripts. When I added `utils` I ensured it was a unique name unused by any packages just to limit the potential effects of my PR. I think compiling scripts is correct, but this PR may have a lot more changes if `et cp --all` was run.

Reply:
FYI several packages already have `scripts` (run `ls packages/*/scripts`).